ena-submission/scripts/ena_submission_helper.py

```diff
@@ -129,6 +129,28 @@ def get_project_xml(project_set):
         "PROJECT": dataclass_to_xml(project_set, root_name="PROJECT_SET"),
     }
 
+def reformat_authors(authors: str) -> str:
+    """This function reformats the Loculus authors string to the format expected by ENA
+    Loculus format: `Doe, John A.; Roe, Jane B. C.;`
+    EMBL expected: `Doe J.A., Roe J.B.C.`
```

Comment:
```suggestion
    EMBL expected: `Doe J.A., Roe J.B.C.;`
```
EMBL also seems to expect authors to end with a `;`

Reply:
They expect yes, but that doesn't mean we need to expect that - their requirements are different as the flat file format is unstructured text as opposed to a TSV. We don't need to expect a trailing `;` - it makes code more complicated as we can't split/join by `;`, also makes things more verbose